browser/src/control/Control.StatusBar.js

```diff
@@ -242,6 +242,7 @@ class StatusBar extends JSDialog.Toolbar {
 			this._generateHtmlItem('permissionmode'),					// spreadsheet, text, presentation
 			{type: 'toolitem', id: 'signstatus', command: '.uno:Signature', w2icon: '', text: _UNO('.uno:Signature'), visible: false},
 			{type: 'spacer',  id: 'permissionspacer'},
+			{type: 'customtoolitem',  id: 'documentstatus'},
```

Comment:
could be `generateHtmlItem` as other widgets here

Reply:
Sure, no problem. But why not all of the items are generated with _generateHtmlItem ? What is the difference ?

Reply:
Because it's just an adapter for old widgets moved from w2ui toolbar to our JSDialog one.
Other widgets are "native" JSDialog's widgets